md-setup/src/main/java/com/wipro/ats/bdre/md/setup/SetupDB.java

```diff
@@ -122,7 +120,8 @@ public static String replaceVars(String line) throws Exception {
     private String[] getColumns(String line) throws Exception {
         if (line.trim().isEmpty() || line.trim().startsWith("--") || line.trim().startsWith("#")) {
             LOGGER.info("Ignoring comment:" + line);
-            return null;
+            String[] nullString=null;
+            return nullString;
         }
         //replace the variables
         line=replaceVars(line);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "line". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)